janitor/functions.py

```diff
@@ -1176,7 +1176,10 @@ def _get_day(x):
 @pf.register_dataframe_method
 @deprecated_alias(column="column_name")
 def filter_column_isin(
-    df: pd.DataFrame, column_name, iterable: Iterable, complement: bool = False
+    df: pd.DataFrame,
+    column_name: str,
+    iterable: Collection,
```

Comment:
The subtleties between `Collection`, `Sequence`, `Iterable`, ... have somewhat eluded me.

Reply:
I'm just serving the mypy overlord :smile: